lib/carrierwave/video/ffmpeg_options.rb

```diff
@@ -76,7 +76,7 @@ def watermark_params
       private
 
         def defaults
-          @defaults ||= { resolution: '640x360', watermark: {} }.tap do |h|
+          @defaults ||= { resolution: @resolution, watermark: {} }.tap do |h|
```

Comment:
The `@resolution` variable is set in the constructor and defaults to `'640x360'` as well.